src/test/components/Timeline.test.js

```diff
@@ -179,15 +179,17 @@ describe('Timeline', function() {
         .mockImplementation(() => ctx);
 
       const store = storeWithProfile(profile);
-      const { getByText } = render(
+      const { getByText, getByRole } = render(
         <Provider store={store}>
           <Timeline />
         </Provider>
       );
 
       expect(getRightClickedTrack(store.getState())).toEqual(null);
 
-      fireEvent.mouseDown(getByText('Process 0'), { button: 2 });
+      fireEvent.mouseDown(getByRole('button', { name: 'Process 0' }), {
+        button: 2,
+      });
```

Comment:
Just wondering if this getByRole is needed here, or is a precursor of your other patches :)

Reply:
There were multiple `Process 0` texts in this patch. I figured saying "get me the button that says Process 0" seemed like a good approach.